pkg/meta/redis.go

```diff
@@ -412,53 +412,6 @@ func (r *redisMeta) parseEntry(buf []byte) (uint8, Ino) {
 	return buf[0], Ino(binary.BigEndian.Uint64(buf[1:]))
 }
 
-func (r *redisMeta) parseAttr(buf []byte, attr *Attr) {
```

Comment:
move this into baseMeta, then you don't need update the caller.

Reply:
ok